src/test/java/api/signup/UpdateUserPositiveTest.java

```diff
@@ -0,0 +1,150 @@
+        sAssert.assertAll();
+    }
+
+    @Test(dataProvider = "username")
+    public void UpdateUserUsernameFields(String updatedUsername) {
+        UpdateUser updateUser = new UpdateUser.Builder()
+                .username(updatedUsername)
+                .build();
+        ResponseWrapper<UserResponse> parsedResponse = ResponseHandler.executeAndGetParsedResponse(
+                authorizedApiService.updateUser(new UpdateUserRequest(updateUser)));
+
+        SoftAssert sAssert = new SoftAssert();
+        sAssert.assertEquals(parsedResponse.getStatusCode(), 200);
+        sAssert.assertEquals(parsedResponse.getResponseBody().getUser().getUsername(), updatedUsername);
+        sAssert.assertAll();
+    }
+
+    @Test(dataProvider = "image")
+    public void UpdateUserOptionalImageFields(String updatedImage) {
+        UpdateUser updateUser = new UpdateUser.Builder()
+                .image(updatedImage)
+                .build();
+        ResponseWrapper<UserResponse> parsedResponse = ResponseHandler.executeAndGetParsedResponse(
+                authorizedApiService.updateUser(new UpdateUserRequest(updateUser)));
+
+        SoftAssert sAssert = new SoftAssert();
+        sAssert.assertEquals(parsedResponse.getStatusCode(), 200);
+        sAssert.assertEquals(parsedResponse.getResponseBody().getUser().getImage(), updatedImage);
+        sAssert.assertAll();
+    }
+
+    @Test(dataProvider = "bio")
+    public void UpdateUserOptionalBioFields(String updatedBio) {
+        UpdateUser updateUser = new UpdateUser.Builder()
+                .bio(updatedBio)
+                .build();
+        ResponseWrapper<UserResponse> parsedResponse = ResponseHandler.executeAndGetParsedResponse(
+                authorizedApiService.updateUser(new UpdateUserRequest(updateUser)));
+
+        SoftAssert sAssert = new SoftAssert();
+        sAssert.assertEquals(parsedResponse.getStatusCode(), 200);
+        sAssert.assertEquals(parsedResponse.getResponseBody().getUser().getBio(), updatedBio);
+        sAssert.assertAll();
+    }
+
+    @Test
+    public void UpdateUsernameFieldTwice() {
+        //TODO add json schema assert
+        String updatedUsername = USER_NAME_PREFIX + System.nanoTime() + System.currentTimeMillis();
+        UpdateUser updateUser = new UpdateUser.Builder()
+                .username(updatedUsername)
+                .build();
+        ResponseHandler.executeAndGetParsedResponse(
+                authorizedApiService.updateUser(new UpdateUserRequest(updateUser)));
+
+        ResponseWrapper<UserResponse> parsedResponse = ResponseHandler.executeAndGetParsedResponse(
+                authorizedApiService.updateUser(new UpdateUserRequest(updateUser)));
+
+        SoftAssert sAssert = new SoftAssert();
+        sAssert.assertEquals(parsedResponse.getStatusCode(), 200);
+        sAssert.assertEquals(parsedResponse.getResponseBody().getUser().getUsername(), updatedUsername);
+        sAssert.assertAll();
+    }
```

Comment:
**suggestion**: add a case to test update of all fields within a single request

Reply:
done